### PR TITLE
Celo tokens phase 2

### PR DESCRIPTION
**Pull request by mateodaza**
Added
 - ETHIX
 - PLASTIK
 - NCT (Toucan Protocol)

### CODE REVIEW

File: src/sdk/sdkV2.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ export default class SdkV2 implements ISdk {
       case CHAIN_ID.MAINNET:
       case CHAIN_ID.ROPSTEN:
       case CHAIN_ID.POLYGON:
+      case CHAIN_ID.CELO:
```
Comment (aminlatifi) [inline src/sdk/sdkV2.ts]: I don't think this line is necessary

Comment (aminlatifi) [reply src/sdk/sdkV2.ts]: I tried to run tests without it, successful

**Review by aminlatifi — CHANGES_REQUESTED** [review]
Just removing that single line, other than that it looks good.

Thanks @mateodaza

File: src/sdk/sdkV2.ts (hunk 1 above)
Comment (mateodaza) [reply src/sdk/sdkV2.ts]: done, thanks man

**Review by aminlatifi — APPROVED** [review]
Thanks @mateodaza

**Pull request merged**